src/sql/workbench/browser/designer/designer.ts

```diff
@@ -855,8 +855,9 @@ export class Designer extends Disposable implements IThemable {
 				});
 				if (taskbar) {
 					taskbar.context = { table: table, path: propertyPath, source: view };
+					this._actionsMap.get(taskbar).map(a => a.table = table);
 				}
-				let buttonColumn: Slick.Column<Slick.SlickData>[] = [];
+				let columns: Slick.Column<Slick.SlickData>[] = [];
```

Comment:
const whenever possible.

Reply:
it's not a const, there are items being pushed in it

Reply:
const just means that the variable isn't going to be reassigned. You can modify stuff in the object all you want (you need to freeze and object to prevent that kind of modification)

Reply:
I am pretty sure you can push items to an array marked as const.